css/style.css

```diff
@@ -0,0 +1,1172 @@
+body {
+  overflow-x: hidden;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  background: url("img/back3.jpg"), linear-gradient(#d3d3d3, #d3d3d3);
+
+  /* W3C */
+  font-family: "Lato", sans-serif;
+}
+
+* {
+  list-style: none;
+}
+
+section {
+  width: 90%;
+  margin: 20px auto;
+}
+
+img {
+  width: 100%;
+  height: auto;
+}
+
+.wrapper {
+  width: 100%;
+  box-shadow: 0 3px 10px rgba(0, 0, 0, 0.2);
+  height: auto;
+  margin: 0 auto;
+  padding: 20px 15px;
+  background-color: #f3f3fa;
+}
+
+header {
+  background-color: #fff;
+  height: 60px;
+}
+
+header nav {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+  width: 80%;
+  margin: 0 auto;
+  padding: 1px 10px;
+}
```

Comment:
- [ ] [Optional] [Highly recommended] Kindly there is a margin that appears on the right side of the nave bar on both home and about pages. Please fix that (See the image in the comment above).

- [ ] [Optional] [Highly recommended] In the desktop view please center the logo and nav links to make your site more beautiful and match the given design.

![image](https://user-images.githubusercontent.com/68796232/169312440-14aeaebb-fb16-4f25-a050-14a61468b187.png)